components/Components/QueryTest.php

```diff
@@ -558,7 +558,7 @@ public static function sameQueryAfterSortingProvider(): array
         return [
             'same already sorted' => ['a=3&a=1&b=2'],
             'empty query' => [null],
-            'contains each pair key only once' => ['batman=robin&aquaman=aqualad&foo=bar&bar=baz'],
+            'contains each pair key only once' => ['aquaman=aqualad&bar=baz&batman=robin&foo=bar'],
```

Comment:
Why did this test change?

Reply:
Query::sort was supposed to follow WHATWG specification. Since now URLSearchParams does just that the sorting was bugfixes hence the change in the tests.

Reply:
Also sorting was wrong in this case its not because the keys appear only once that they should not be sorted.